content/en/docs/rosenpass-tool/guides/_index.html

```diff
@@ -0,0 +1,107 @@
+---
+aliases: [/docs/start, /docs/rosenpass-tool/start]
+title: "Quick Start: Installation and Usage Guides"
+linkTitle: "Quick Start"
+subtitle: "Build post-quantum-secure VPNs with WireGuard!"
+weight: 2
+hideLead: true
+type: docs
+menu: false
+noList: true
+shortBlerb: "Installation and Usage Guides for Rosenpass"
+blerb: "A quick start guide on downloading, configuring, and using the Rosenpass tool together with a WireGuard connection."
+---
+
+{{% blocks/lead color="secondary" %}}
+## Quick Start: Installation and Usage Guides
+{{% /blocks/lead %}}
+
+{{% blocks/section color="light" class="no-flex contains-code-snippets" %}}
+
+### Supported Systems
+
+We provide packages for **Nix**, **Arch Linux**, and **Alpine Linux**, and dedicated guides for manual installation on **[Ubuntu](ubuntu)** and **[Debian](debian)**.
+
+Since **Rosenpass** was released not too long ago, **packaging is an ongoing process**. You can find the latest information about all the Linux distributions we support on **[Repology](https://repology.org/project/rosenpass/versions")**.
+
+Even if your distribution is not listed here, you can always **compile Rosenpass yourself** or download a pre-built, statically linked **binary** from our [GitHub release page](https://github.com/rosenpass/rosenpass/releases/latest). You can also find OCI container images (Docker, Podman, etc.) there. While we only offer x86_64 builds, there's no principle reason limiting **Rosenpass** to x86_64, and you can compile it for any architecture in Linux, or MacOS, that is supported by [liboqs](https://openquantumsafe.org/liboqs/#overview).
```

Comment:
```suggestion
Even if your distribution is not listed here, you can always **compile Rosenpass yourself** or download a pre-built, statically linked **binary** from our [GitHub release page](https://github.com/rosenpass/rosenpass/releases/latest). You can also find OCI container images (Docker, Podman, etc.) there. While we only offer x86_64 builds, there's no technical reason limiting **Rosenpass** to x86_64. You can compile it for any architecture that is supported by [liboqs](https://openquantumsafe.org/liboqs/#overview) and capable of running Linux (or macOS).
```

(Note: I don't know if this is actually true - I assume all of the libraries etc. are working on other platforms)